src/server/game/Entities/Pet/Pet.cpp

```diff
@@ -1418,6 +1361,10 @@ bool Pet::addSpell(uint32 spellId, ActiveStates active /*= ACT_DECIDE*/, PetSpel
 
     m_spells[spellId] = newspell;
 
+    // Do not go further if this Attribute exists
+    if (spellInfo->HasAttribute(SPELL_ATTR4_UNK15))
```

Comment:
Gonna need pointers to client code to accept that

Reply:
https://gist.github.com/mdX7/7d5b0369896f6e88eeb3ce83bc2b7a02#file-cpp-L37

Reply:
https://gist.github.com/mdX7/f6e1b7857737517b4565324aae0c98a7#file-cpp-L57

Reply:
https://gist.github.com/mdX7/136a1c6ea349c5e9a1902ab4d656ce61#file-cpp-L1617

Reply:
https://gist.github.com/mdX7/2d7bda471c5c1bb239ab3d3791f0540a#file-cpp-L1258

Reply:
https://gist.github.com/mdX7/8f17acb234b7fc0b2016c79bce33818b#file-cpp-L190

Thanks to @mdX7

Reply:
These spells do not appear into learned spells list of pet also